app/screens/permalink/permalink_error.tsx

```diff
@@ -131,7 +128,6 @@ function PermalinkError({
                     theme={theme}
                     value={text}
                     baseTextStyle={style.errorText}
-                    baseParagraphStyle={style.errorTextParagraph}
```

Comment:
Why?

Reply:
This is already centered with the `errorText` style.